packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -131,13 +152,24 @@ export function PanelChrome({
 
   const testid = title ? selectors.components.Panels.Panel.title(title) : 'Panel';
 
+  const collapsibleHeader = (
+    <button type="button" className={styles.clearButtonStyles} onClick={toggleOpen}>
+      <Icon name={isOpen ? 'angle-down' : 'angle-right'} />
```

Comment:
this icon misses a textual representation, given the panel title can be an empty string we may want to consider adding a label to this icon if the title is not set, or aria-hidden otherwhise. something like:

```suggestion
      <Icon
        name={isOpen ? 'angle-down' : 'angle-right'}
        aria-hidden={!!title}
        aria-label={!title ? 'toggle collapse panel' : undefined}
      />
```